dropwizard-client/src/main/java/io/dropwizard/client/HttpClientBuilder.java

```diff
@@ -355,9 +355,9 @@ protected ConfiguredCloseableHttpClient createClient(
             final InstrumentedHttpClientConnectionManager manager,
             final String name) {
         final String cookiePolicy = configuration.isCookiesEnabled() ? StandardCookieSpec.RELAXED : StandardCookieSpec.IGNORE;
-        final Integer timeout = (int) configuration.getTimeout().toMilliseconds();
-        final Integer connectionTimeout = (int) configuration.getConnectionTimeout().toMilliseconds();
-        final Integer connectionRequestTimeout = (int) configuration.getConnectionRequestTimeout().toMilliseconds();
+        final int timeout = (int) configuration.getTimeout().toMilliseconds();
```

Comment:
Why should be use primitives here now?

Reply:
I think that it did unnecessary boxing.
```java
final Integer timeout = (int) configuration.getTimeout().toMilliseconds();
```

Reply:
Sorry. I thought the builder methods would require boxed types. Then it won't made any difference, where the boxing is performed.